lib/tasks/categorize_cafe.rake

```diff
@@ -8,5 +8,12 @@ namespace :db do
       end
       puts "Categorized all cafes!"
     end
+
+    desc 'Adds cafes categories to the beginning of Street Cafe names that are categorized as medium and large'
+    task update_names: :environment do
+      cafes = StreetCafe.cafes_by_category('%medium', '%large') 
```

Comment:
I decided to just use the database to batch update these cafes rather than iterating with a poro